README.md

```diff
@@ -1188,40 +1188,40 @@ Use this parameter — especially when using the `fuzzy_match` parameter — to
 
 * **instance_name**
 
-  The name of the instance to start a job on. The value defaults to the default instance on the machine.
+  The name of the instance. The value defaults to the instance on the machine.
```

Comment:
The second `default` here needs to stay. The default instance is the instance you get if you only install one instance of sqlserver on a machine. If you install two instances you get the default instances, and then you get a named instance, or if you like you could install two named instances. But the default one is usually there and referred to by just the machine name. Other non-default instances have to be referred to by a specific name.